features/bsm/persistence/api/pom.xml

```diff
@@ -38,5 +38,14 @@
       <groupId>com.google.guava</groupId>
       <artifactId>guava</artifactId>
     </dependency>
+    <dependency>
+      <groupId>javax.validation</groupId>
+      <artifactId>validation-api</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.hibernate</groupId>
+      <artifactId>hibernate-validator</artifactId>
+      <version>4.3.2.Final</version>
```

Comment:
The hibernate-validator is used some place else in bsm. Please define a `hibernateValidatorVersion` property in the root pom and re-use the property everywhere else please.